bin/cli.js

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+const testRunner = require('../src/');
+const argv = require('yargs')
+    .usage("Usage: $0 <option> test file")
```

Comment:
This usage string doesn't seem to match what yargs actually enforces – yargs expects `roca -f /path/to/file` (or `roca --file /path/to/file`, but the usage string seems to expect `roca /path/to/file`.  To get positional arguments like that in yargs, you'll need something like:

```suggestion
#!/usr/bin/env node
const testRunner = require('../src/');
const argv = require('yargs')
    .usage("Usage: $0 [options] <file>")
    .updateStrings({
        "Positionals:": "Positional Arguments",
        "Options:": "Other Options"
    })
    .command(
        "* <file>",
        false, // keep this comand hidden from --help
        (yargs) => {
            yargs.positional("file", {
                type: "string",
                describe: "The test file to execute"
            })
        },
        (argv) => {
            testRunner(argv.file);
        }
    )
    .help("h")
    .alias("h", "help")
    .argv;
```

The benefit is that yargs will do the `if (argv.f)` checking for you and automatically show the help output if the correct arguments weren't provided!

(disclosure: I copied the `.updateStrings` bit directly [from mocha](https://github.com/mochajs/mocha/blob/b5c0fb0a26ee9f8fd574e78732a2cf715d7ab6ca/lib/cli/cli.js#L39-L43) 😉 )